src/app/account-app/account-upgrades.component.html

```diff
@@ -1,59 +1,119 @@
 <style>
+@media(min-width: 1024px) {
+    .limitedTimeOffer, .productCardSpecial {
+	margin: 2% 25%;
+    }
+}
+
+@media(max-width: 1023px) {
+    .limitedTimeOffer, .productCardSpecial {
+	margin: 2% 10%;
+    }
+}
+
+@media(max-width: 813px) {
+    .limitedTimeOffer, .productCardSpecial {
+	margin: 2% 0;
+    }
+}
+
+.limitedTimeOffer h2 {
+    white-space: nowrap;
+}
+
+mat-icon.limitedTimeOffer {
+    margin: 0;
+    width: auto;
+    border: none;
+    color: primary;
+    font-size: 40px;
+    line-height: 12px;
+}
+
 #productGrid {
-	display: flex;
-	flex-flow: row wrap;
-	justify-content: space-between;
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-between;
 }
 
 .productCard {
-	margin: 10px;
-	flex: 450px 1 0;
+    margin: 10px;
+    flex: 200px 1 0;
+}
+
+.productCardHide {
+    display: none;
 }
+
 .productCardSpecial {
-        margin: 10px;
-        flex: 900px 1 0;
-        text-align: center;
+    text-align: center;
+}
+
+
+.productCardSpecialHide {
+    display: none;
 }
 </style>
 
-<h2> Account upgrades </h2>
+<h2> Account plans </h2>
 
 <div *ngIf="rmmapi.me | async as me; else loading">
     <p>
         <strong> Acccount: {{ me.first_name }} {{ me.last_name }} ({{ me.user_address }}) </strong>
     </p>
 
     <p>
-      Here you can change your account's subscription plan.
+      Here you can subscribe to Runbox, or change an existing subscription plan according to your needs.
     <p>
-      Note: When selecting a subscription plan, please note that your current email disk usage is
+      Note: When selecting a subscription plan for your main account, please note that your current email disk usage is
       {{ (me.disk_used / 1024 / 1024 / 1024) | number:'1.2-2' }} GB.
     </p>
 
-    <p>
-        And don't forget to check out
+    <div *ngIf="!(future_date < now)" class="limitedTimeOffer">
```

Comment:
Both `future_date` and `now` don't exist as variables, turning this into `!(undefined < undefined)` (`true`) and thus making it always appear.